isotpsend.c

```diff
@@ -87,6 +89,8 @@ int main(int argc, char **argv)
     static struct can_isotp_ll_options llopts;
     int opt;
     extern int optind, opterr, optopt;
+    unsigned int loops = 1; /* one (== no) loop by default */
+    unsigned int usecs = 0; /* wait before sending the PDU */
```

Comment:
My man page says `usleep()` uses a `useconds_t` as argument.

Reply:
Yes and `useconds_t` is `unsigned long`. I will put `useconds_t` here and let `strtoul()` complain if it doesn't fit ;-)